e2e-tests-rust/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "era_test_node_e2e_tests"
+edition = "2021"
+authors = ["The Matter Labs Team <hello@matterlabs.dev>"]
+homepage = "https://zksync.io/"
+repository = "https://github.com/matter-labs/era-test-node"
+license = "MIT OR Apache-2.0"
+keywords = ["blockchain", "zksync"]
+categories = ["cryptography"]
+publish = false
+
+[dependencies]
+anyhow = "1.0"
+fs2 = "0.4.3"
+tokio = { version = "1", features = ["time", "rt", "process"] }
+
+[dev-dependencies]
+alloy-zksync = { git = "https://github.com/itegulov/alloy-zksync.git", rev = "b433d2d64e0f7d939891064b00d8a7991a427682" }
```

Comment:
Using my own branch for now as this needs https://github.com/popzxc/alloy-zksync/pull/28 to be upstreamed